lib/poxa/channels_handler.ex

```diff
@@ -22,11 +22,13 @@ defmodule Poxa.ChannelsHandler do
   def malformed_request(req, _state) do
     {info, req} = :cowboy_req.qs_val("info", req, "")
     attributes = String.split(info, ",")
+      |> Enum.reject(fn s -> s == "" end)
```

Comment:
I would write: `Enum.reject(&(&1 == ""))` but that's just cosmetic. Whatever you may prefer :+1: